system/ee/templates/_themes/forum/blue/forum_announcements/announcement.html

```diff
@@ -25,7 +25,12 @@
 <td>{lang:posted_on} {post_date format="%d %F %Y %h:%i %A"}</td>
 <td align="right">&nbsp;
 {if can_view_ip}<i>{lang:ip_address} {ip_address}</i>{/if}
-{if can_change_status}&nbsp;&nbsp;<span class="smallLinks">[ <a href="{path:change_status}">{lang:change_status}</a> ]</span>{/if}
+{if can_change_status}&nbsp;&nbsp;
+	<form action="{path:change_status}" method="POST">
+		<input type="hidden" name="csrf_token" value = {csrf_token}>
+		<input type="submit" name="submit" value = "{lang:change_status}">
```

Comment:
I have noticed that in other places of forum all submit buttons also have `class="submit"` - should this be applied here as well?

Also, is there a reason for inconsistency of spaces around ` = `?

This comment is applicable to the other themes as well